scripts/minikube.sh

```diff
@@ -202,6 +211,14 @@ up)
 
     disable_storage_addons
 
+    #  get kubernetes version we are operating on and accordingly enable feature gates
+    KUBE_MAJOR=$(kube_version 1)
+    KUBE_MINOR=$(kube_version 2)
+    if  [ "${KUBE_MAJOR}" -eq 1 ] && [ "${KUBE_MINOR}" -ge 20 ];
+    then
+    # if kubernetes version is greater than 1.20 enable Generic Ephemeral Volume feature gate
+    K8S_FEATURE_GATES="GenericEphemeralVolume=true"
```

Comment:
`K8S_FEATURE_GATES` is already set at line 163, this overrides that option. Should the variable not be extended with the feature gate instead?

Reply:
my plan was to remove the expandCSIVolumes feature gate as a follow up as it no longer wanted, will file a seperate PR for the same, however its better to be in appended string form here, so that in future if there is already set value, it should carry that in final string.